marshmallow/validate.py

```diff
@@ -465,3 +465,48 @@ def __call__(self, value):
                 del choices[index]
 
         return value
+
+
+class Or(Validator):
+    """Validator which succeeds if ``value`` is validated by at least one of
+    ``validator_list`` elements
+
+    :param validator_list:
+    :param str error: Error message to raise in case of a validation error. Can be
+        interpolated with `{input}`.
+    """
+
+    default_message = "{input!r} can't be validated by any of the validators"
+
+    def __init__(self, *validator_list, error=None, **kwargs):
+        if any(not isinstance(validator, Validator) for validator in
+                validator_list):
```

Comment:
The visually better way to split into lines is to put expression at first line and "for .. " on second:
```python
if not all(isinstance(validator, Validator)
           for validator in validator_list):